test/fixtures/experimental/class-private-properties/static/actual.js

```diff
@@ -0,0 +1,4 @@
+class A {
+  static #x;
+  static #y = 1;
+}
```

Comment:
Have comma-separated field declarations landed yet? If so, it'd be nice to have tests that demonstrate that you can mix static public and private field declarations in a comma-separated list.